templating/embedding_controllers.rst

```diff
@@ -67,8 +67,9 @@ The ``recent_list`` template is perfectly straightforward:
     (e.g. ``/article/*slug*``). This is a bad practice. In the next section,
     you'll learn how to do this correctly.
 
-To include the controller, you'll need to refer to it using the standard
-string syntax for controllers (i.e. **bundle**:**controller**:**action**):
+To include the controller, you'll need to refer to it using the standard string syntax for controllers (i.e. **controllerPath**:**action**).
```

Comment:
Actually, I believe we should use the `::` syntax. A single colon *requires* the controller to be a service. But if you use `::`, then Symfony looks in the container first to see if it is a service, but then falls back to instantiating the controller. So, it's a bit more universal.